rest/client/methods.go

```diff
@@ -313,8 +313,35 @@ func (c *communicatorImpl) SendLogMessages(ctx context.Context, taskData TaskDat
 	}
 	info.setTaskPathSuffix("log")
 	var cancel context.CancelFunc
-	ctx, cancel = context.WithTimeout(ctx, 10*time.Minute)
+	ctx, cancel = context.WithDeadline(ctx, time.Now().Add(10*time.Minute))
```

Comment:
I don't think this is necessary.

Reply:
I don't think so either, but Eric Daniels suggested changing this for some reason

Reply:
`WithTimeout` is implemented like this. I think Eric's point was that if you use `WithDeadline` you can log `time.Now()`, which will help diagnose a problem with the clock if there is one.

Reply:
then we should be logging that? this is just more oblique.

Reply:
Sort of. If you,
```
now := time.Now()
// log
ctx, cancel = context.WithDeadline(now, <etc>
```
Then you're more sure it's the same now.

Reply:
added a log statement